src/components/Results.vue

```diff
@@ -1,71 +1,79 @@
 <template>
-  <div v-if="q && data && data">
+  <div v-if="q && data">
     <div
       v-for="dataset in data.terms"
       :key="dataset.source.uri"
       class="container"
     >
       <h2 class="display-5 text-center py-3">
         {{ dataset.source.name }}
-        <small class="termCount">({{ t('search.termsFound', dataset.terms.length) }})</small>
+        <small v-if="dataset.source.alternateName">({{ dataset.source.alternateName }})</small>
+        <small class="termCount">: {{ t('search.termsFound', dataset.terms.length) }}</small>
+        <p>
+          <small class="termCount">{{ dataset.source.creators[0].name }}</small>
```

Comment:
If a source doesn't have an alternate name, the `<h2>` contains a space between the name and the term count. For example:

`Adamlink: straten in Amsterdam : 1 term` or 
`Adamlink: straten in Amsterdam : geen termen`.

Reply:
Good point, will fix.

Another detail that doesn’t look too well is when the source name already has a colon (`:`) in it. In your wireframes the number of results per source was missing, but I think we still want to show that. One idea would be to show the it in a `<small>` element.

Reply:
Yes, I like the idea of showing the term count in a separate element. The current title (name of dataset + alternate name, if any + term count) is a bit too much, I think. Also, it isn't really a 'title', because of its dynamic nature (i.e. the term count). Perhaps we could use a fixed title (name of dataset + alternate name) and put the term count somewhere else?

I've omitted the term count from the wireframes because of the new 'lookup' function. A lookup can result in 'no term found' or in 'one term found'. Do you think we should show the term count in this case, too?

Reply:
Agreed: for the lookup it doesn’t make sense to show any count. However, I’d like to finish search first before switching to lookup. For search it’s nice to keep the count.

Related: with the vertical cards there’s much more scrolling involved, so users may easily overlook results in other sources if the first source has many results. Perhaps some overview at the top of the results with all sources and (linking to that source’s results on the page) and place the term count there as well?

Reply:
Made the term count a bit more subtle now.